core/tests/org.openjdk.jmc.flightrecorder.serializers.test/src/test/java/org/openjdk/jmc/flightrecorder/serializers/json/test/IItemCollectionJsonSerializerTest.java

```diff
@@ -66,7 +66,7 @@ public void testSerializeKnownRecording() throws IOException {
 
 		String actual = IItemCollectionJsonSerializer.toJsonString(testRecording);
 
-		assertEquals(expected, actual);
+		assertEquals(expected.replaceAll("\\r\\n", "\n"), actual.replaceAll("\\r\\n", "\n"));
```

Comment:
Sorry, I'm late to the party 🙈 I wrote this test and I'm curious what the problem was here 😃 

Did this test fail for you? I don't understand how it can be different on windows since the JSON file has unix line endings and the serialiser will output unix line endings.

Reply:
Hm, it's been a while but obviously it failed for me. In the bug I've written: On Windows, the returned data from IItemCollectionJsonSerializer.toJsonString(testRecording) seems to have platform line endings. Are you sure that there should be unix line endings?

Reply:
:+1: Yup, you're right. Sorry, I didn't check [`StructuredWriter.java`](https://github.com/openjdk/jmc/blob/master/core/org.openjdk.jmc.flightrecorder.serializers/src/main/java/org/openjdk/jmc/flightrecorder/serializers/json/StructuredWriter.java#L61), there we are using `String.format("%n");` which will output platform dependent line terminators 😃